pagopa/pn-frontend
# error message for denomination #pn-3046

## Short description
Two different error message for denomination with length over 80 characters and with forbidden characters

## List of changes proposed in this pull request
- Changed validation in Recipient.tsx file

## How to test
- Login with pa -> go to new notification -> add denomination with length greater than 80 characters -> check both name and username are highlighted and that an error message is shown
- Login with pa -> go to new notification -> add a denomination with forbidden characters -> check that only the field with wrong char is highlighted and that an error message is shown
